docs/ai-workspace/configuration.md

```
@@ -87,13 +87,26 @@ org_handle_claim_name   = "org_handle"
-| Platform API env variable | Description |
-|--------------------------|-------------|
-| `AUTH_JWT_SECRET_KEY` | JWT signing key (required when `auth.jwt.enabled = true`) |
-| `ENCRYPTION_KEY` | 32-byte key (64 hex / base64) — encrypts secrets & subscription tokens (required) |
-| `DATABASE_PASSWORD` | Database password |
+```toml
+encryption_key = '{{ env "APIP_CP_ENCRYPTION_KEY" }}'            # from an env var
+secret_key     = '{{ file "/secrets/platform-api/jwt_secret" }}' # from a file (preferred)
+```
+
+Supply the env values from a git-ignored `keys.env` and start with `docker compose --env-file keys.env up
+Or mount a secret file and use `{{ file }}`.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)